src/misc/lv_mem.c

```diff
@@ -204,6 +204,39 @@ void * lv_mem_realloc(void * data_p, size_t new_size)
     return new_p;
 }
 
+bool lv_mem_realloc_safe(void ** data, size_t new_size)
+{
+    void * new_p = NULL;
+
+    MEM_TRACE("reallocating %p with %lu size", *data, (unsigned long)new_size);
+
+    if((*data == NULL) && (0 < new_size)) {
```

Comment:
`size_t` is always unsigned so it can't be negative.
https://stackoverflow.com/questions/1089176/is-size-t-always-unsigned

Reply:
Will replace with a `0 != new_size` check

Reply:
It cant be negative but can be zero. Check seems legit unless there are other code style rules by lib.